app/views/facilitator/outreach_events/_outreach_events_list.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="table-container">
```

Comment:
We may want to consider consolidating all of the `_outreach_events_list` partials into one "smart" partial that conditionally outputs fields depending on the user type. We don't necessarily need to do it now, but it may help us in the long run since maintaining three separate files will become quite tedious.